internal/server/node/triple.go

```diff
@@ -272,19 +272,40 @@ func applyLimit(dcid string, t *pb.Triples, limit int) {
 				isOut = "1"
 				neighborTypes = t.ObjectTypes
 			}
-
-			for _, nt := range neighborTypes {
-				key := isOut + t.Predicate + nt
-				if _, ok := existTriple[key]; !ok {
-					existTriple[key] = []*pb.Triple{}
-				}
-				existTriple[key] = append(existTriple[key], t)
+			var nt string
```

Comment:
should this choice of nt be deterministic?

Reply:
the neighbor type is deterministic now, either "" for value, or the first type of the corresponding entity.